test/e2e/specs/specs-wpcom/wp-calypso-gutenberg-page-editor-tracking-spec.js

```diff
@@ -40,10 +40,16 @@ describe( `[${ host }] Calypso Gutenberg Page Editor Tracking: (${ screenSize })
 			await wpAdminSidebar.selectNewPage();
 
 			const editor = await GutenbergEditorComponent.Expect( this.driver, 'wp-admin' );
-			await editor.initEditor( { dismissPageTemplateLocator: false } );
+			await editor.initEditor( { dismissPageTemplateLocator: true } );
```

Comment:
Remove this line and above

Reply:
Im not sure I follow, if we remove this here wont the editor never close the block sidebar.  Meaning when we call `initEditor` later without the page templates dismissed it will get stuck trying to close the sidebar sitting behind the modal?

Reply:
ah I see what you are getting at!